tests/test_bias.py

```diff
@@ -280,6 +280,35 @@ def test_bias_separate_damping():
     assert algo.user_offsets_.loc[[10, 12, 13]].values == \
         approx(np.array([0.266234, -0.08333, -0.22727]), abs=1.0e-4)
 
+def test_transform_user_with_user_bias():
+    algo = Bias()
+    algo.fit(simple_df)
+
+    new_ratings = pd.Series([4.0, 5.0], index=[1, 2]) # items as index and ratings as values
+
+    ratings_with_bias, user_bias = algo.transform_user(new_ratings) # user: 13
+    ratings_without_bias = algo.inverse_transform_user(13, ratings_with_bias, user_bias)
+
+    assert new_ratings[1] == ratings_without_bias[1]
+    assert new_ratings[2] == ratings_without_bias[2]
+
+def test_transform_user_without_user_bias():
+    user = 12
+    algo = Bias()
+    algo.fit(simple_df)
+
+    new_ratings = pd.Series([-0.5, 1.5], index=[2, 3]) # items as index and ratings as values
+
+    ratings_without_bias = algo.inverse_transform_user(user, new_ratings)
+
+    r_no_b_0 = ratings_without_bias[2]
+    r_u_i_m_0 = new_ratings[2] + algo.user_offsets_.loc[user] + algo.item_offsets_.loc[2] + algo.mean_
+
+    r_no_b_1 = ratings_without_bias[3]
+    r_u_i_m_1 = new_ratings[3] + algo.user_offsets_.loc[user] + algo.item_offsets_.loc[3] + algo.mean_
+
+    assert r_no_b_0 == r_u_i_m_0
+    assert r_no_b_1 == r_u_i_m_1
```

Comment:
let's write these as:

```python
assert ratings_without_bias[2] == new_ratings[2] + algo.user_offsets_.loc[user] + algo.item_offsets_.loc[2] + algo.mean_
```

If it's too long, we can rename `ratings_without_bias` to `result` or `v` or something.

That way, if the assertion fails, PyTest will give us more information about what went wrong.